.gemspec

```
@@ -38,7 +38,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency(%q<locale>, "~> 2.1")
   s.add_runtime_dependency(%q<multi_json>, "~> 1.13")
   s.add_runtime_dependency(%q<httpclient>, "~> 2.8")
-  s.add_runtime_dependency(%q<concurrent-ruby>, "~> 1.0")
+  s.add_runtime_dependency(%q<concurrent-ruby>, "~> 1.1.0")
```

Technically, 1.0 still work, so we do not need to bump the minimal required version.  Might be safer to only add an upper bound:

```suggestion
  s.add_runtime_dependency(%q<concurrent-ruby>, ["~> 1.0", "< 1.2.0"])
```

---

yeah that's true, I'll update that